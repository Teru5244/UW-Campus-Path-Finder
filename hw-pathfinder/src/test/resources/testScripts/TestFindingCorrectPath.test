# Test if the searching algorithm is working properly.

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 1.0
AddEdge g n2 n3 1.0
AddEdge g n3 n4 1.0
AddEdge g n1 n3 1.0

FindPath g n1 n3

FindPath g n1 n4