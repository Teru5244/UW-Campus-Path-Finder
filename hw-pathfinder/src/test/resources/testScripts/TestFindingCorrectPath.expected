# Test if the searching algorithm is working properly.

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge 1.000 from n1 to n2 in g
added edge 1.000 from n2 to n3 in g
added edge 1.000 from n3 to n4 in g
added edge 1.000 from n1 to n3 in g

path from n1 to n3:
n1 to n3 with weight 1.000
total cost: 1.000

path from n1 to n4:
n1 to n3 with weight 1.000
n3 to n4 with weight 1.000
total cost: 2.000